core/src/main/resources/defaults/blueprints/7.2.11/cdp-data-engineering-ha.bp

```diff
@@ -551,7 +551,9 @@
         "refName": "manager",
         "cardinality": 1,
```

Comment:
Should this be changed to 2, to avoid confusion?

How many of these instances actually exist? Hue has been added on masterx, and manager. Is there 1 or >1 instance of the manager role?

Reply:
I never understood the meaning of cardinality. At the moment it is treated as minimum node count, but I have not seen any failure directly given if I give less than the cardinality in the cluster definition. The manager is 1 in GCP, 2 in AWS and Azure for 7.2.11. For node count, we should treat the cluster definition as the source of truth.

Reply:
So this ends up with 3 instances of Hue? 2 on the manager nodes, and 1 on masterx?
OT: How does CB handle 2 nodes with CM on it. (Will ping offline)

Reply:
3 instances of Hue for AWS and Azure. CM generates a warning that the recommended configuration is only max 2 load-balancers. We can revisit this warning later.